src/EnrollPasswordController.js

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'util/FormController',
+  'util/ValidationUtil',
+  'util/FactorUtil',
+  'views/enroll-factors/Footer',
+  'views/shared/TextBox'
+],
+function (Okta, FormController, ValidationUtil, FactorUtil, Footer, TextBox) {
+
+  var _ = Okta._;
+
+  return FormController.extend({
+    className: 'enroll-password',
+    Model: {
+      props: {
+        password: ['string', true],
+        confirmPassword: ['string', true]
+      },
+      validate: function () {
+        return ValidationUtil.validateFieldsMatch(this, 'password', 'confirmPassword', 
+          Okta.loc('password.enroll.error.match', 'login'));
+      },
+      save: function () {
+        return this.doTransaction(function (transaction) {
+          var factor = _.findWhere(transaction.factors, {
+            factorType: 'password',
+            provider: 'OKTA'
+          });
+          return factor.enroll({
+            profile: {
+              password: this.get('password')
+            }
+          });
+        });
+      }
+    },
+
+    Form: {
+      autoSave: true,
+      title: 'enroll.password.setup',
```

Comment:
we should use `_.partial` here. This is because we load other languages on demand, so having a function here guarantees that the label will be correct
```
title: _.partial(Okta.loc, 'enroll.password.setup', 'login'),
```

Reply:
My mistake, did some hard coding of the message for testing purposes and accidentally committed that. Should be fixed now.